src/message_broadcaster/http.rs

```diff
@@ -54,3 +58,38 @@ impl Clone for HttpMessageBroadcaster {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Was reading the code and thought why not add a test. I think this is pretty neat if you want to test http code.

Reply:
Good thinking. Always can use more tests.